xml/ay_custom_user_scripts.xml

```diff
@@ -613,10 +619,10 @@ ls
       &lt;/source&gt;
     &lt;/script&gt;
     &lt;script&gt;
-      &lt;filename&gt;chroot.sh&lt;/filename&gt;
-        &lt;interpreter&gt;shell&lt;/interpreter&gt;
+      &lt;filename&gt;chroot-pre.sh&lt;/filename&gt;
+        &lt;interpreter&gt;/bin/bash -x&lt;/interpreter&gt;
         &lt;source&gt;&lt;![CDATA[
-#!/bin/sh
+#!/bin/bash
```

Comment:
Note the CDATA content starts with a newline, which makes the `#!` useless. Also, YaST would ignore it even if it were in the right place. Let's delete it.

(not specific to this \<source>, all sources in this example have this problem)

Reply:
agreed